src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -72,6 +72,9 @@ let PUT     = httpVerb "PUT"    : HttpHandler
 let PATCH   = httpVerb "PATCH"  : HttpHandler
 let DELETE  = httpVerb "DELETE" : HttpHandler
 
+[<Literal>]
+let private RouteKey = "route"
```

Comment:
Let's set the value to something which is less likely to conflict with other code which might set an item called "route". Perhaps something like "aspnet_lambda_route" ?